Index: mapogr.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- mapogr.cpp	(revision fb13d41e69e3e1be24ec13d36a933553b89bfe55)
+++ mapogr.cpp	(date 1572434066981)
@@ -3497,7 +3497,7 @@
                                           pSpatialFilterNode);
   }
 
-  if( expr->m_nToken == MS_TOKEN_COMPARISON_INTERSECTS &&
+  if( (expr->m_nToken == MS_TOKEN_COMPARISON_INTERSECTS || expr->m_nToken == MS_TOKEN_COMPARISON_CONTAINS) &&
       expr->m_aoChildren.size() == 2 &&
       expr->m_aoChildren[1]->m_nToken == MS_TOKEN_LITERAL_SHAPE )
   {
